modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -7268,6 +7271,7 @@ field.user-id=User
 field.user-name=User Name
 fields=Fields
 fields-are-added-if-you-need-to-distinguish-items-based-on-criteria-chosen-by-the-user=Fields are added if you need to distinguish items based on criteria chosen by the user. For example, if you are selling t-shirts, you may want to add a field with the name <em>Size</em> and the values <em>S, M, L, XL</em>. Enter the values as a list of comma delimited options.
+fields-cannot-be-deleted-from-unique-composite-keys-after-object-publication=Fields cannot be deleted from Unique Composite Keys after Object publication.
```

Comment:
@pedro-oliveira446 @mateussgn Please change the `Object publication` to `definition publication` (or, `after publishing the definition.`). `Object` can be anything.

If we don't add the `Object` prefix to `fields`, for example, it should be `definition publication`. 

Please see more details in this [thread](https://liferay.slack.com/archives/C026UR5976G/p1695930050190819).